src/cuga/backend/server/a2a/router.py

```
@@ -0,0 +1,182 @@
+
+        if method == "message/send":
+            try:
+                params = MessageSendParams.model_validate(params_dict)
+            except Exception as exc:
+                return JSONResponse(_rpc_error(rpc_id, _INVALID_PARAMS, "Invalid params", str(exc)))
+            message_text = _extract_message_text(params)
+            context_id = _ensure_context_id(params)
+            task_id = uuid.uuid4().hex
+            try:
+                task = await _run_and_collect(message_text, context_id, task_id)
+            except Exception as exc:  # graph blew up
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)